app/services/metrics/taxonomy_count.rb

```diff
@@ -0,0 +1,13 @@
+module Metrics
+  class TaxonomyCount
+    def self.count_taggings_per_level
+      counts_by_level = Taxonomy::TaxonomyQuery.new.taxons_per_level.map do |level|
```

Comment:
"level" is also used below, and even though its in a different scope, the meaning is quite different. I think "taxons" could work here as a name?